plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/KubernetesClusterManagerImpl.java

```diff
@@ -644,7 +652,15 @@ public KubernetesClusterResponse createKubernetesClusterResponse(long kubernetes
         return response;
     }
 
+    private void validateEndpointUrl() {
+        String csUrl = ApiServiceConfiguration.ApiServletPath.value();
+        if (csUrl == null || csUrl.contains("localhost")) {
+            throw new InvalidParameterValueException("Global setting endpointe.url has to be set to the Management Server's API end point");
```

Comment:
You may want to fix the typo?

Reply:
Nope, it's a global variable

Reply:
What about `ApiServiceConfiguration.ApiServletPath.key()` instead of `endpointe.url`?

Reply:
Would be better to retrieve as @nvazquez said. Thus, if/when this gets fixed it will automatically reflect in this message.